<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaOpciones</class>
 <widget class="QMainWindow" name="VentanaOpciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>262</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>455</width>
    <height>262</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>455</width>
    <height>262</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Opciones</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelEmailUsuario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelContrasena">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEmailUsuario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditContrasena">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelServidor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Servidor:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditServidor">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxServidor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Elegir el Servidor</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPuerto">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Puerto:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPuerto">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhFormattedNumbersOnly</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxContrasena">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar contraseña</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGuardar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCancelar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>455</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
